semgrep-core/src/core/Rule.ml

```diff
@@ -316,7 +316,7 @@ let kind_of_formula = function
 (* Converters *)
 (*****************************************************************************)
 
-(* Substitutes `$MVAR` with `int($MVAR)` in cond. *)
+(* Substitutes `$MVAR` with `int(digits($MVAR))` in cond. *)
```

Comment:
This preserves the meaning of `int` but it breaks the spec of the `strip:` key. I would avoid that too. You could perhaps add a different key. (Although I personally think `metavariable-comparison` should just take a Python expression and nothing else, but that's just my opinion anyways.)